.github/workflows/ci.yml

```
@@ -100,26 +100,16 @@ jobs:
 
   build-docker-arm64:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
 
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: arm64
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
+      - name: Build Docker
+        run: docker compose -f docker-compose-arm64.yml build --build-arg release=0
```

I think we can get rid of the `--build-arg` for both the arm64 and amd64 build since the Dockerfiles only compile Invidious in release mode when `release` is preset and is set as 1

```suggestion
        run: docker compose -f docker-compose-arm64.yml build
```